test-cases/unrestricted-transfer-from/unrestricted-transfer-from-1/remediated-example/psp22/Cargo.toml

```
@@ -0,0 +1,31 @@
+[package]
+name = "psp22_base_contract"
+version = "0.1.0"
+authors = ["[your_name] <[your_email]>"]
+edition = "2021"
+
+[dependencies]
+# Import ink!
+ink = { version = "4.2.1", default-features = false }
+# Brush dependency
```

Unnecessary comments
```suggestion
ink = { version = "4.2.1", default-features = false }
```